src/main/java/edu/harvard/iq/dataverse/dataverse/banners/dto/BannerMapper.java

```diff
@@ -109,7 +110,13 @@ private DataverseLocalizedBanner mapToLocalizedBanner(DataverseBanner banner, Da
         DataverseLocalizedBanner dataverseLocalizedBanner = new DataverseLocalizedBanner();
 
         if (isBannerReused(fuDto)) {
```

Comment:
Do we need to rely on what is in `fuDto.getFile()`? Is there is some situation when `fuDto.getDisplayedImage()` will be something different than `fuDto.getFile()`?